packages/plugin-lighthouse/CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Contributing
+
+## Setup
+
+Make sure to install dependencies:
+
+```sh
+npm install
```

Comment:
👩‍💻DX: Chrome should be added as a prerequisite here as well.

Reply:
I did not do that on purpose. The packages inside this repo should include Chromium. 

Could you please check the stup and tell me how it could happen that the packages did not install chromium? 
If this is really the case we should put the into the `packages.json`.

Reply:
Where can I find the dependency that should include chromium? In `package.json`, I do not see Chromium.

Reply:
when you install lighthouse is has to run on something... what got installed on your end?
Also, can we clarify this in slack and merge it?

Reply:
I have Chrome installed in a different location than the hardcoded one. That is why it did not work for me.

But my original point was something else. If we only provide Lighthouse but the browser it runs on **needs to be installed separately** by the user, it should be listed as a **prerequisite** for using this plugin in its documentation. If we install a browser and run Lighthouse on it for the user, then the documentation is okay as is.